tests/bud.bats

```diff
@@ -5325,35 +5383,35 @@ _EOF
 @test "bud-with-mount-bind-from-relative-like-buildkit" {
   skip_if_no_runtime
   skip_if_in_container
-  mkdir ${TEST_SCRATCH_DIR}/bud
-  cp -R $BUDFILES/buildkit-mount-from ${TEST_SCRATCH_DIR}/bud/buildkit-mount-from
+  local contextdir=${TEST_SCRATCH_DIR}/buildkit-mount-from
+  cp -R $BUDFILES/buildkit-mount-from $contextdir
   # build base image which we will use as our `from`
-  run_buildah build -t buildkitbaserelative $WITH_POLICY_JSON -f ${TEST_SCRATCH_DIR}/bud/buildkit-mount-from/Dockerfilebuildkitbaserelative ${TEST_SCRATCH_DIR}/bud/buildkit-mount-from/
+  run_buildah build -t buildkitbaserelative $WITH_POLICY_JSON -f $contextdir/Dockerfilebuildkitbaserelative $contextdir/
   # try reading something from image in a different build
-  run_buildah build -t testbud $WITH_POLICY_JSON -f ${TEST_SCRATCH_DIR}/bud/buildkit-mount-from/Dockerfilebindfromrelative
+  run_buildah build -t testbud $WITH_POLICY_JSON -f $contextdir/Dockerfilebindfromrelative
   expect_output --substring "hello"
   run_buildah rmi -f buildkitbaserelative
   run_buildah rmi -f testbud
 }
 
 @test "bud-with-mount-bind-from-multistage-relative-like-buildkit" {
-  mkdir ${TEST_SCRATCH_DIR}/bud
-  cp -R $BUDFILES/buildkit-mount-from ${TEST_SCRATCH_DIR}/bud/buildkit-mount-from
+  local contextdir=${TEST_SCRATCH_DIR}/buildkit-mount-from
```

Comment:
Yes. This is part of the second-part commit, the hand-tuning one. There were a bunch of these, and the gist of all of them is:
```diff
-   mkdir       x/bud
-   cp -R [...] x/bud/subdir
+   cp -R [...] x/subdir
```
Reason: the only thing the `mkdir` was doing was making a subdir so `cp -R` could then write into a sub-subdir. There's no reason for that, it's perfectly fine to use just one subdir. Since `cp -R` will create one level of directory, I removed the `bud` entirely. Difficult diff to read, granted, but much better for the resulting code base. Your second comment is exactly the same situation.